libs/blocks/section-metadata/section-metadata.css

```diff
@@ -19,6 +19,7 @@
 
 .section.static-links a:not([class]) {
   color: inherit;
+  text-decoration: underline;
```

Comment:
As @rgclayton mentioned, there should definitely be a hover indicator, whatever that may be from a Consonant perspective. This will also come up during an accessibility review, a visual indicator should exist between states.